config/forms/2021_2022.json

```diff
@@ -4308,10 +4308,10 @@
                       "value": "Internal transfer"
                     },
                     "2": {
-                      "value": "Tenant applied directly (no referral or nomination)"
+                      "value": "Tenant applied directly (no nomination)"
                     },
                     "3": {
-                      "value": "Private registered provider (PRP) lettings only – nominated by a local housing authority"
+                      "value": "Nominated by a local housing authority"
```

Comment:
@roobottom Do we need to include the text ‘Private registered provider (PRP) lettings only’ given the option is only shown in this case anyway?